### DISCUSSION TITLE
any demos on robot visualization?

### DISCUSSION THREAD

**Discussion by HongminWu** [dfki-ric/pytransform3d#151] [Q&A] [answered]
Would you kindly provide an example for showing the Kuka-iiwa robot as described in the main page?

**Comment by AlexanderFabisch** [comment, marked as answer]
Here is an example with another URDF: https://rock-learning.github.io/pytransform3d/_auto_examples/visualizations/vis_moving_robot.html#sphx-glr-auto-examples-visualizations-vis-moving-robot-py
